README.md

```diff
@@ -1,9 +1,10 @@
-# ISIS3
+# ISIS
```

Comment:
We should go through and scrub the entire Readme of ISIS3 references at once. I'd rather not do this piece-meal.

Reply:
I going to revert this and we can put in a different PR to do that.